Source/WebCore/animation/KeyframeEffect.h

```diff
@@ -132,9 +132,9 @@ class KeyframeEffect : public AnimationEffect, public CSSPropertyBlendingClient
 
     void willChangeRenderer();
 
-    Document* document() const override;
-    RenderElement* renderer() const override;
-    const RenderStyle& currentStyle() const override;
```

Comment:
Why are we removing the `override` ? 

`override final` would be safer

Reply:
I won't argue for or against `override final` but there is no use of `override final` in the entire WebKit project. I don't think we have a policy to use either `override` or `final` in classes marked `final`, I was only making that change for consistency within that file which uses `final` instead of `override` in the vast majority of cases.